modules/auxiliary/scanner/smb/smb_version.rb

```diff
@@ -189,10 +193,21 @@ def smb_os_description(res, nd_smb_fingerprint)
   # Fingerprint a single host
   #
   def run_host(ip)
-    smb_ports = [445, 139]
```

Comment:
Not a blocker; looks like `modules/auxiliary/scanner/smb/smb_uninit_cred.rb` uses this pattern too

Reply:
Yes, I found a few other modules using this pattern and called them out in the description. If we like this solution, we can apply it to the others as well so they offer consistent functionality.